plugins/CoreAdminHome/lang/en.json

```diff
@@ -24,7 +24,7 @@
         "JavaScriptTracking": "JavaScript Tracking",
         "JSTracking_CampaignKwdParam": "Campaign Keyword parameter",
         "JSTracking_CampaignNameParam": "Campaign Name parameter",
-        "JSTracking_CodeNote": "Make sure this code is on every page of your website before the %1$s tag.",
+        "JSTracking_CodeNote": "Make sure this code is on every page of your website within the %1$s section, or just after the %2$s tag.",
```

Comment:
Should we maybe write as early in the &lt;head&gt; as possible? Eg if many other scripts are loaded in the head before the Piwik tag, users might still get javascript errors because of not defined `_paq.push`. 

It's very common to shrink and merge all JS files into one file. If this file is placed before `piwik.js` there is a good chance that `_paq` might be not defined yet. 

Also wondering if we should write something like "... to prevent possible JavaScript errors with not defined `_paq.push`..." to explain why it is good to place it as early in `head` element as possible